### PR TITLE
difference-of-squares: adjust API to use instance method [Fix #253]

### LINKED ISSUE #253: Grade-school: Structure very unclear
The intended structure of the School class is very hard to pin down. All of the add() calls are done directly on the School, but then for the first few tests, things (size, added students) are accessed through an arbitrary middle layer, `db()`. After the first few tests, suddenly the calls change from `school.db().get({grade number})` to `school.grade({grade number})`.

Examples:
    `school.add("Chelsea", 3);`
    `assertThat(school.db().get(3).size(), is(1));`
    `assertThat(school.grade(5).size(), is(2));`

If the point of the exercise is to force adherence to an arbitrary (and frankly ridiculous) structure, then I would consider it a great success.

### PR DESCRIPTION
**Pull request by FridaTveit**
Fixes issue #253.

### CODE REVIEW

File: exercises/difference-of-squares/src/example/java/DifferenceOfSquaresCalculator.java [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
-    public static int computeSumOfSquaresTo(final int input) {
+    public int computeSumOfSquaresTo(final int input) {
         return IntStream.rangeClosed(1, input)
                 .map(i -> (int) Math.pow(i, 2))
                 .sum();
     }
 
-    public static int betweenSquareOfSumAndSumOfSquaresTo(final int input) {
+    public int betweenSquareOfSumAndSumOfSquaresTo(final int input) {
```
Comment (stkent) [inline exercises/difference-of-squares/src/example/java/DifferenceOfSquaresCalculator.java]: Given the rename of the test class, maybe updating this to `computeDifferenceOfSquares` would be clearer and shorter now?

**Comment by stkent** [comment]
@FridaTveit one tiny suggestion, looks good to me otherwise!

File: exercises/difference-of-squares/src/example/java/DifferenceOfSquaresCalculator.java (hunk 1 above)
Comment (FridaTveit) [inline exercises/difference-of-squares/src/example/java/DifferenceOfSquaresCalculator.java]: True, will change it! :)

**Comment by stkent** [comment]
Great, thanks @FridaTveit!

**Pull request merged**